src/modules/storage/LedgerStorage.ts

```diff
@@ -2257,6 +2320,22 @@ export class LedgerStorage extends Storages
         return this.query(sql, [from, to]);
     }
 
+    /**
+    * Get Average Fees between given time range.
+    * @param from Begin date for chart history 
+    * @param to End date for chart history
+    */
+    public calculateAvgFeeChart (from: number, to: number): Promise<any[]> {
+        let sql =
+            `SELECT
+                height, time_stamp, average_tx_fee
+             FROM
+                fees
+             WHERE
+                time_stamp BETWEEN ? AND  ? `;
```

Comment:
When querying a wide range (annual) of data, a lot of data can occur.

Reply:
we have to response with whole data within that range. i don't think pagination is needed here.

Reply:
I think there is enough room for improvement here.
Please prepare improvement PR next time.